src/plugins/maxlength/plugin.yml

```diff
@@ -0,0 +1,13 @@
+about: |
+  Use dmenu for input of maximum length, without 
+  any menu items. Acts as a replacement for `i3-input -l`
+  Pass --maxlength=<MAXLENGTH> to use
```

Comment:
> without any menu items

Is the intent to never have any menu items on screen? If so, you can completely disable them by pasting the following at the bottom of this plugin's `main.rs`:
```rust
use crate::config::ConfigDefault;
#[override_flag(flag = maxlength)]
impl ConfigDefault {
    pub fn nostdin() -> bool {
	true
    }
}
```

This also removes the need to `echo ''|dmenu --maxlength X`, instead having `dmenu --maxlength X`.